### PR TITLE
PAE-499 - Add VUE login

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description:
---------------
This PR changes the design of the login

Screenshots:
-----------------

image

**PR links**: [PAE-339](https://pearsonadvance.atlassian.net/browse/PAE-339), [PAE-499](https://pearsonadvance.atlassian.net/browse/PAE-499)

Reviewers:
---------------
- @diegomillan 
- @Squirrel18 
- @ivanvgh

### CODE REVIEW

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 1]
```diff
@@ -1,7 +1,358 @@
-// Extra rules needed for an LMS theme
+// Extra rules needed for the LMS dark theme
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: A period at the end of the line. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 2]
```diff
@@ -1,7 +1,358 @@
...
+@import 'profile';
+@import 'index';
+@import 'dashboard';
+@import 'support';
+@import 'course-experience';
+
+
+//registration
+.view-register .form-actions button[type="submit"]:hover {
+  background-color:#fff;
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: Please add a space before the value assignment throughout the file. @luismorenolopera 
`background-color: #fff;`

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 3]
```diff
@@ -1,7 +1,358 @@
...
+  border: 1px solid #007FA3;
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: Can we get this value from a variable definition? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 4]
```diff
@@ -1,7 +1,358 @@
...
+}
+
+
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: There are two empty lines between the class definitions in the whole file, can we leave a single line? Instead of two ... @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 5]
```diff
@@ -1,7 +1,358 @@
...
+  right: 110px;
+  position: absolute;
+  top: 100px;
+}
+
+.signin-container {
+  background: white;
+  padding: 30px;
+  border-radius: 10%;
+  box-shadow: 3px 2px 2px 5px #bcb9b9;
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: Can we get the colour value from a variable? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 6]
```diff
@@ -1,7 +1,358 @@
...
+}
+
+.signing-title {
+  text-align: center;
+  font-size: 18px;
+  font-weight: 600;
+  margin-bottom: 10px;
+}
+
+.login-register .form-wrapper .login-providers {
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:274]: Can we get the colour values from a variable? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 7]
```diff
@@ -1,7 +1,358 @@
...
+  margin-top: 15px;
+  text-align: center;
+}
+.login-register .action-primary {
+  width: 100%;
+  display: block;
+}
+
+.background-login {
+  background: url('/static/pearson-vue-theme/images/CertPREP-Sign-In-background.jpg');
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:323]: Would this need to be changed via /admin? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 8]
```diff
@@ -1,7 +1,358 @@
...
+  height: 100vh;
+  width: 100%;
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:325]: Why not 100vw? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 9]
```diff
@@ -1,7 +1,358 @@
...
+    background-image: none;
+    background-color: $primary;
+    border-color: $primary;
+}
+
+@media (min-width: 768px) {
+  .content-wrapper {
+     padding: 0 0 10px;
+  }
+}
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: Can you arrange this in a media query section, at the bottom of the file? including the other media query definitions... @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss [hunk 10]
```diff
@@ -497,7 +497,7 @@ $homepage__header--gradient__color--bravo: saturate($gray, 30%) !default;
 $homepage__header--background: lighten($gray, 15%) !default;
 $homepage-background: rgb(252, 252, 252) !default;
 $course-card-height: ($baseline*18) !default;
-$course-image-height: ($baseline*8) !default;
+$course-image-height: auto;
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss:500]: Would this affect all other images on the platform? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables.scss [hunk 11]
```diff
@@ -1,3 +1,113 @@
-// Default bootstrap theming
+// Sample dark theme to demonstrate SASS overrides
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables.scss]: A period at the end of the line. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/student-account/login.underscore [hunk 12]
```diff
@@ -1,80 +1,70 @@
-<div class="js-form-feedback" aria-live="assertive" tabindex="-1">
+<div class="background-login"></div>
+
+<div>
+  <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/student-account/login.underscore:4]: Would this need to be changed via /admin? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/student-account/login.underscore [hunk 13]
```diff
@@ -1,80 +1,70 @@
...
-                <span class="text"><%- gettext("or sign in with") %></span>
-            </h3>
-        </div>
-    <% } %>
+      <div class="section-title lines">
+        <h3>
+          <span class="text"><%- gettext("or sign in with") %></span>
+        </h3>
+      </div>
+      <% } %>
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/student-account/login.underscore]: This additional indentation is not neccesary. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html [hunk 14]
```diff
@@ -1,56 +1,4 @@
-<div class="js-form-feedback" aria-live="assertive" tabindex="-1">
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html]: Why was this block removed? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html [hunk 15]
```diff
@@ -107,29 +75,8 @@ <h3>
...
-                    </div>
-                    <span aria-hidden="true"><%- provider.name %></span>
-                    <span class="sr"><%- _.sprintf( gettext("Sign in with %(providerName)s"), {providerName: provider.name} ) %></span>
-                </button>
-            <% }
-        }); %>
-    </div>
-    <% } %>
-</form>
+<script type="text/javascript">
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html:78]: Why is this necessary? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html [hunk 16]
```diff
@@ -107,29 +75,8 @@ <h3>
 </div>
 
 
-    <button type="submit" class="action action-primary action-update js-login login-button"><%- gettext("Sign in") %></button>
-
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html]: Why was this block removed? @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss [hunk 17]
```diff
@@ -1,7 +1,358 @@
...
+  color: $primary-font-color;
+}
+
+.field-input{
+  background-color: $inverse;
+}
+
+.btn {
+  border-color: $inverse;
+  background: $primary ;
```
Comment (ivanvgh) [inline edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: Can we leave this without the space before the semicolon and keep a space before the value assignment for every property? @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/student-account/login.underscore [hunk 18]
```diff
@@ -1,80 +1,70 @@
...
-                            <img class="icon-image" src="<%- provider.iconImage %>" alt="<%- provider.name %> icon" />
-                        <% } %>
-                    </div>
-                    <span aria-hidden="true"><%- provider.name %></span>
-                    <span class="sr"><%- _.sprintf( gettext("Sign in with %(providerName)s"), {providerName: provider.name} ) %></span>
-                </button>
-            <% }
-        }); %>
-    </div>
+        <% _.each( context.providers, function( provider ) {1
```
Comment (ivanvgh) [inline edx-platform/pearson-vue-theme/lms/student-account/login.underscore]: Could you remove the number 1 at the end of this line?. It seems not to have a purpose. @luismorenolopera

**Review by ivanvgh — CHANGES_REQUESTED** [review]
@luismorenolopera Could you review those comments I left please?

File: edx-platform/pearson-vue-theme/lms/student-account/login.underscore (hunk 12 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/student-account/login.underscore:4]: @Squirrel18 maybe, but it is not the intention of this PR

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss (hunk 7 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:323]: @Squirrel18  is it possible to get a change via admin using sass?

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss (hunk 8 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:325]: @Squirrel18 It was required to see an image and a text in the lower part, so the size was modified vertically

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss (hunk 10 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss:500]: yes

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html (hunk 14 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html]: move to login.underscore

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html (hunk 16 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html]: move to login.underscore

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html (hunk 15 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html:78]: to change the title when clicking on the change password modal @Squirrel18

File: edx-platform/pearson-vue-theme/lms/student-account/login.underscore (hunk 18 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/student-account/login.underscore]: ok

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss (hunk 6 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:274]: done

**Review by ivanvgh — APPROVED** [review]
Thanks.

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss (hunk 9 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss]: done

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss (hunk 7 above)
Comment (Squirrel18) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:323]: Am just asking about the possibility of change this value from site configuration. Do you think will it be necessary to change from site settings?

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss (hunk 10 above)
Comment (Squirrel18) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss:500]: Is that the desired behaviour?

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html (hunk 15 above)
Comment (Squirrel18) [reply edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html:78]: Is it required for this? Is there any other way to do it?

Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html:78]: The page does not change when it is clicked, because it is a modal of the login page, so it cannot be changed using the block for this @Squirrel18

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss (hunk 7 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss:323]: I do not think so

File: edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html (hunk 15 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html:78]: By block, I mean:

`<%block name="pagetitle">${_("Sign in or Register")}</%block>`

File: edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss (hunk 10 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_variables-v1.scss:500]: yes

**Comment by luismorenolopera** [comment]
@ivanvgh @Squirrel18 I have added 3 very nice images in this PR

**Review by Squirrel18 — APPROVED** [review]
